### PR TITLE
fix(ui): Issues around Add Data Quality Test #7498

### LINKED ISSUE #7498: UI: Issues around Add Data Quality Test
- [ ] "Create new test suite" button and the section should not be displayed if the user does not have permission to create a new test suite
Screen Shot 2022-09-15 at 8 55 56 AM

- [ ] Once the "Create new test suite" button is clicked, there is no way to go back to select an already created test suite. The cancel button takes the user back to the entity page.

Screen Shot 2022-09-15 at 8 58 21 AM

- [ ] There is no active cursor displayed when the description box is selected. This feedback is critical to improve user experience. Ideally the placeholder text should be replaced with an active cursor when the description box is selected.

https://user-images.githubusercontent.com/1051198/190451816-03cfd10b-ac78-43c0-b1d5-93430aaa2041.mov

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on the Fixing UI: Issues around Add Data Quality Test
- Only the admin can add a test suite
- Active cursor onFocus of description
- Removed disabled state of select test suite dropdown onclick of add new test suite
Closes #7498 

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/71748675/191276650-81abb006-bd4a-4d01-a5a7-8948031d5cd6.mov

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by chirag-madlani — COMMENTED** [review]
Looks good to me. Can we remove tailwind classnames please?

**Comment by ShaileshParmar11** [comment]
> Looks good to me. Can we remove tailwind classnames please?

I have updated the code, Can you have a look?

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/SelectTestSuite.tsx [hunk 1]
```diff
@@ -92,99 +110,125 @@ const SelectTestSuite: React.FC<SelectTestSuiteProps> = ({
...
+                    margin: 0,
+                  }}
+                  onTextChange={() => {
+                    resetSelectedId();
+                  }}
+                />
+              </Form.Item>
+            </>
+          ) : (
+            <Row className="m-b-xlg" justify="center">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/SelectTestSuite.tsx:216]: Row must contain col & not any other element as child

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 2]
```diff
@@ -14,6 +14,12 @@
 @import '~antd/dist/antd.less';
 @import url('./variables.less');
 
+//lg: 24px;
+//md: 16px;
+//sm: 12px;
+//xs: 8px;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:20]: Thanks for this one 👍

**Review by chirag-madlani — APPROVED** [review]
Left one minor comment but rest looks good to me

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**